### DISCUSSION TITLE
How do i remove close icon from tour?

### DISCUSSION THREAD

**Discussion by ajitesh-kuppa-sivakumar** [gilbarbara/react-joyride#801] [Q&A] [answered]
<img width="474" alt="Screenshot 2022-05-11 at 7 15 05 PM" src="https://user-images.githubusercontent.com/40975704/167864717-570b5320-386a-43a9-b681-f410ce7b5013.png">
Hi People,

I want to remove the 'X' (close) icon from tour. I was going through docs and was unable to find (or might have missed) props to do so.
Please assist me on this.

Thanks.

**Comment by gilbarbara** [comment, marked as answer]
hey @ajitesh-sivakumar 

It's described on the [props](https://docs.react-joyride.com/props) page,  `hideCloseButton`.
